contracts/JetStakingV2.sol

```diff
@@ -964,7 +1051,7 @@ contract JetStakingV1 is AdminControlled {
         uint256 streamId,
         uint256 start,
         uint256 end
```

Comment:
in this function `rewardsSchedule` we can use memory to store `Schedule` as it does not update the storage. This is optional and if done, there should there should be test cases for max array values stored in the struct. Recommending it as this function is used in other external functions and would reduce gas cost if memory variable is used.

Reply:
If possible, it could be extended to all getters :)

Reply:
Referencing your comment as an issue.